bindings/ros/aditof_roscpp/src/examples/rviz_pointcloud.cpp

```diff
@@ -37,12 +37,15 @@ using namespace aditof;
 
 int main(int argc, char **argv) {
 
-    std::shared_ptr<Camera> camera = initCameraEthernet(argc, argv);
+    std::shared_ptr<Camera> camera = initCamera(argc, argv);
     if (camera.get() == nullptr) {
         ROS_ERROR("initCamera call failed");
         return 0;
```

Comment:
Usually if return is 0, that means is with success, but you call an error before. in my opinion it is not consistent.